tests/test_constraints.py

```diff
@@ -15,7 +15,7 @@
 # Exclude certain operators for solvers.
 # Not all solvers support all operators in CPMpy
 EXCLUDE_OPERATORS = {"gurobi": {"mod"},
-                     "pysat": {"sum", "wsum", "sub", "mod", "div", "pow", "abs", "mul","-"}}
```

Comment:
I wil make some adjustments here. Pysat does not have to support all `sum's` and `wsums` right? Only sums and wsums where args are bool.